modules/labs-react/search-form/lib/SearchForm.tsx

```diff
@@ -118,173 +117,318 @@ function getInputColors(theme: SearchThemeAttributes, isFocused?: boolean) {
   };
 }
 
-const formCollapsedBackground = colors.frenchVanilla100;
+const formCollapsedBackground = '#fff';
```

Comment:
This is used for `chroma` and I can't have a css variable. I'm not actually styling the button, it's just used in `getIconButtonType` to determine if it's inverse or not.